app/views/comments/reply_to_comment.js.erb

```diff
@@ -0,0 +1,14 @@
+commentParentDiv = $("#comment-<%= @comment.parent_id %>");
+replyDiv = commentParentDiv.children('.reply');
```

Comment:
the main difference between replying to a comment or a reference is the reply div wrapper and seeing if it exists already.